crypto/hash/hmac_ossl.c

```diff
@@ -101,11 +174,25 @@ static srtp_err_status_t srtp_hmac_alloc(srtp_auth_t **a,
 
 static srtp_err_status_t srtp_hmac_dealloc(srtp_auth_t *a)
 {
-    HMAC_CTX *hmac_ctx;
-
-    hmac_ctx = (HMAC_CTX *)a->state;
+    srtp_hmac_ossl_ctx_t *hmac = (srtp_hmac_ossl_ctx_t *)a->state;
+
+    if (hmac) {
+#ifdef SRTP_OSSL_USE_EVP_MAC
+        if (hmac->ctx != NULL) {
+            EVP_MAC_CTX_free(hmac->ctx);
+        }
+        if (hmac->ctx_dup != NULL) {
+            EVP_MAC_CTX_free(hmac->ctx_dup);
```

Comment:
[The documentation](https://www.openssl.org/docs/man3.0/man3/EVP_MAC_CTX_free.html) says "NULL is a valid parameter, for which this function is a no-op.", so you don't need the `if` branch here.